processor/schemaprocessor/transformer_test.go

```diff
@@ -30,17 +29,6 @@ import (
 	"go.uber.org/zap/zaptest"
 )
 
-//go:embed testdata/schema.yml
-var schemaContent []byte
-
-func SchemaHandler(t *testing.T) func(wr http.ResponseWriter, r *http.Request) {
-	assert.NotEmpty(t, schemaContent, "SchemaContent MUST not be empty")
-	return func(wr http.ResponseWriter, r *http.Request) {
-		_, err := wr.Write(schemaContent)
-		assert.NoError(t, err, "Must not have issues writing schema content")
-	}
-}
-
```

Comment:
Is this deleted intentionally? No longer needed?

Reply:
I will need to double check, may have accidentally  caught it in git.

Reply:
This is intentional , it will be moved to translations in the next PR.